packages/@o3r/eslint-config-otter/index.js

```diff
@@ -1,567 +1,15 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 /* eslint-disable quote-props */
-
 module.exports = {
   'extends': [
-    'eslint:recommended',
-    'plugin:jsdoc/recommended',
-    'plugin:@angular-eslint/recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
-    'plugin:@o3r/recommended',
+    './base',
     'plugin:jest/recommended'
```

Comment:
Then this should be part of the optional peer deps to not be install by people using /base